### ISSUE TITLE
Game crashed on minecraft 1.16.4 after editing a treated wooden pole "The game crashed whilst tesselating block model Error: java.lang.ArrayIndexOutOfBoundsException: 8313 Exit Code: -1"

### ISSUE THREAD

**Issue by HoovySimulator2** [BluSunrize/ImmersiveEngineering#4561]
### Description of the issue:
Sometimes game crashes when a block is being placed, usually the pole blocks or when hammering the top part of a pole. This also affects placing modded blocks from other mods. This happened with Decorative Blocks mod by Stohun and the evidence that this is related to IE is that the crash log contained the name. I have left a report of this to that mod's Issues page with that crash log.

Also the world cannot be rejoined as it crashes immediately after the block causing the crash is loaded. Quick workaround is to remove the mod the block is from, replace it with other blocks in world and install the mod back.

### Crashlog:
Time: 1/10/21 12:06 AM
Description: Tesselating block model

java.lang.ArrayIndexOutOfBoundsException: 8313
	at net.minecraft.client.renderer.chunk.ChunkRenderCache.func_180495_p(SourceFile:103) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.common.blocks.generic.PostBlock.hasConnection(PostBlock.java:313) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.common.blocks.generic.PostBlock.getConnectionShapes(PostBlock.java:242) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.common.blocks.generic.PostBlock.func_220053_a(PostBlock.java:199) ~[?:?] {re:classloading}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_215700_a(AbstractBlock.java:535) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_196954_c(AbstractBlock.java:531) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.block.AbstractBlock.func_196247_c(AbstractBlock.java:226) ~[?:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_235754_c_(AbstractBlock.java:427) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_200015_d(AbstractBlock.java:517) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at blusunrize.immersiveengineering.common.blocks.IEBaseBlock.func_200011_d(IEBaseBlock.java:144) ~[?:?] {re:classloading}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_200016_a(AbstractBlock.java:419) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.client.renderer.BlockModelRenderer$AmbientOcclusionFace.func_239285_a_(BlockModelRenderer.java:306) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.BlockModelRenderer.func_228799_a_(BlockModelRenderer.java:121) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.BlockModelRenderer.renderModelSmooth(BlockModelRenderer.java:75) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.model.pipeline.ForgeBlockModelRenderer.renderModelSmooth(ForgeBlockModelRenderer.java:89) ~[?:?] {re:classloading}
	at net.minecraft.client.renderer.BlockModelRenderer.renderModel(BlockModelRenderer.java:51) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderModel(BlockRendererDispatcher.java:63) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender$RebuildTask.func_228940_a_(ChunkRenderDispatcher.java:527) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender$RebuildTask.func_225618_a_(ChunkRenderDispatcher.java:449) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender.func_228936_k_(ChunkRenderDispatcher.java:383) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher.func_228902_a_(ChunkRenderDispatcher.java:164) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.WorldRenderer.func_228437_a_(WorldRenderer.java:842) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.WorldRenderer.func_228426_a_(WorldRenderer.java:931) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.func_228378_a_(GameRenderer.java:607) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:totaldarkness.mixins.json:MixinGameRenderer,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:425) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:totaldarkness.mixins.json:MixinGameRenderer,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:976) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.4-35.1.32.jar:35.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$473/1810970264.call(Unknown Source) [forge-1.16.4-35.1.32.jar:35.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?] {}

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.client.renderer.chunk.ChunkRenderCache.func_180495_p(SourceFile:103) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.common.blocks.generic.PostBlock.hasConnection(PostBlock.java:313) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.common.blocks.generic.PostBlock.getConnectionShapes(PostBlock.java:242) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.common.blocks.generic.PostBlock.func_220053_a(PostBlock.java:199) ~[?:?] {re:classloading}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_215700_a(AbstractBlock.java:535) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_196954_c(AbstractBlock.java:531) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.block.AbstractBlock.func_196247_c(AbstractBlock.java:226) ~[?:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_235754_c_(AbstractBlock.java:427) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_200015_d(AbstractBlock.java:517) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at blusunrize.immersiveengineering.common.blocks.IEBaseBlock.func_200011_d(IEBaseBlock.java:144) ~[?:?] {re:classloading}
	at net.minecraft.block.AbstractBlock$AbstractBlockState.func_200016_a(AbstractBlock.java:419) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:immersiveengineering.mixins.json:coremods.AbstractBlockStateMixin,pl:mixin:A}
	at net.minecraft.client.renderer.BlockModelRenderer$AmbientOcclusionFace.func_239285_a_(BlockModelRenderer.java:306) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.BlockModelRenderer.func_228799_a_(BlockModelRenderer.java:121) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.BlockModelRenderer.renderModelSmooth(BlockModelRenderer.java:75) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraftforge.client.model.pipeline.ForgeBlockModelRenderer.renderModelSmooth(ForgeBlockModelRenderer.java:89) ~[?:?] {re:classloading}
-- Block model being tesselated --
Details:
	Block: Block{minecraft:snow}[layers=1]
	Block location: World: (203,76,239), Chunk: (at 11,4,15 in 12,14; contains blocks 192,0,224 to 207,255,239), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Using AO: true
Stacktrace:
	at net.minecraft.client.renderer.BlockModelRenderer.renderModel(BlockModelRenderer.java:51) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}

-- Block being tesselated --
Details:
	Block: Block{minecraft:snow}[layers=1]
	Block location: World: (203,76,239), Chunk: (at 11,4,15 in 12,14; contains blocks 192,0,224 to 207,255,239), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
Stacktrace:
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderModel(BlockRendererDispatcher.java:63) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender$RebuildTask.func_228940_a_(ChunkRenderDispatcher.java:527) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender$RebuildTask.func_225618_a_(ChunkRenderDispatcher.java:449) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender.func_228936_k_(ChunkRenderDispatcher.java:383) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher.func_228902_a_(ChunkRenderDispatcher.java:164) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.WorldRenderer.func_228437_a_(WorldRenderer.java:842) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.WorldRenderer.func_228426_a_(WorldRenderer.java:931) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.func_228378_a_(GameRenderer.java:607) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:totaldarkness.mixins.json:MixinGameRenderer,pl:mixin:A,pl:runtimedistcleaner:A}

-- Affected level --
Details:
	All players: 1 total; [ClientPlayerEntity['HoovySimulator2'/241, l='ClientLevel', x=202.60, y=80.07, z=239.22]]
	Chunk stats: Client Chunk Cache: 2209, 1785
	Level dimension: minecraft:overworld
	Level spawn location: World: (249,78,-110), Chunk: (at 9,4,2 in 15,-7; contains blocks 240,0,-112 to 255,255,-97), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Level time: 69098 game time, 85902 day time
	Server brand: forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.world.ClientWorld.func_72914_a(ClientWorld.java:447) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,xf:fml:xaerominimap:xaero_clientworldclass,xf:fml:xaeroworldmap:xaero_wm_clientworldclass,pl:mixin:APP:mixins.sndctrl.json:MixinClientWorld,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2024) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:623) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.4-35.1.32.jar:35.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$473/1810970264.call(Unknown Source) [forge-1.16.4-35.1.32.jar:35.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?] {}

### Versions & Modlist

-- System Details --
Details:
	Minecraft Version: 1.16.4
	Minecraft Version ID: 1.16.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1487869264 bytes (1418 MB) / 4265082880 bytes (4067 MB) up to 4265082880 bytes (4067 MB)
	CPUs: 4
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4096m -Xms256m -XX:PermSize=256m
	ModLauncher: 8.0.6+85+master.325de55
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-3.0.5-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.4-35.1.32.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.4-35.1.32.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.4-35.1.32.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.4-35.1.32.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/forge-1.16.4-35.1.32.jar fml TRANSFORMATIONSERVICE 
	FML: 35.1
	Forge: net.minecraftforge:35.1.32
	FML Language Providers: 
		javafml@35.1
		minecraft@1
	Mod List: 
		forge-1.16.4-35.1.32-client.jar                   |Minecraft                     |minecraft                     |1.16.4              |DONE      |NOSIGNATURE
		mcw-bridges-1.0.4-mc1.16.4.jar                    |Macaw's Bridges               |mcwbridges                    |1.0.4               |DONE      |NOSIGNATURE
		ImmersiveEngineering-1.16.4-4.1.2-129.jar         |Immersive Engineering         |immersiveengineering          |1.16.4-4.1.2-129    |DONE      |44:39:94:cf:1d:8c:be:3c:7f:a9:ee:f4:1e:63:a5:ac:61:f9:c2:87:d5:5b:d9:d6:8c:b5:3e:96:5d:8e:3f:b7
		BiomesOPlenty-1.16.4-13.0.0.427-universal.jar     |Biomes O' Plenty              |biomesoplenty                 |1.16.4-13.0.0.427   |DONE      |NOSIGNATURE
		mining-helmet-1.16.4-2.0.0.jar                    |Mining Helmet                 |mining_helmet                 |2.0.0               |DONE      |NOSIGNATURE
		mcw-trapdors-1.0.0-mc1.16.4.jar                   |Macaw's Trapdoors             |mcwtrpdoors                   |1.0.0               |DONE      |NOSIGNATURE
		jei-1.16.4-7.6.1.65.jar                           |Just Enough Items             |jei                           |7.6.1.65            |DONE      |NOSIGNATURE
		simplefarming-1.16.4-1.3.5.jar                    |Simple Farming                |simplefarming                 |1.16.4-1.3.5        |DONE      |NOSIGNATURE
		DoggyTalents-1.16.4-2.0.1.2.jar                   |Doggy Talents 2               |doggytalents                  |2.0.1.2             |DONE      |NOSIGNATURE
		mcw-windows-1.0.2-mc1.16.4.jar                    |Macaw's Windows               |mcwwindows                    |1.0.2               |DONE      |NOSIGNATURE
		Patchouli-1.16.4-48.jar                           |Patchouli                     |patchouli                     |1.16.4-48           |DONE      |NOSIGNATURE
		create-mc1.16.3_v0.3c.jar                         |Create                        |create                        |mc1.16.3_v0.3c      |DONE      |NOSIGNATURE
		Xaeros_Minimap_20.30.1_Forge_1.16.4.jar           |Xaero's Minimap               |xaerominimap                  |20.30.1             |DONE      |NOSIGNATURE
		totaldarkness-1.2-1.16.3.jar                      |Total Darkness                |totaldarkness                 |1.2-1.16.3          |DONE      |NOSIGNATURE
		XaerosWorldMap_1.11.6_Forge_1.16.4.jar            |Xaero's World Map             |xaeroworldmap                 |1.11.6              |DONE      |NOSIGNATURE
		comforts-forge-1.16.4-4.0.1.0.jar                 |Comforts                      |comforts                      |1.16.4-4.0.1.0      |DONE      |NOSIGNATURE
		TravelersBackpack-1.16.4-5.3.4.jar                |Traveler's Backpack           |travelersbackpack             |5.3.4               |DONE      |NOSIGNATURE
		sit-1.16.2-v1.3.jar                               |Sit                           |sit                           |v1.3                |DONE      |NOSIGNATURE
		decorative_blocks-1.16.4-1.7.0.jar                |Decorative Blocks             |decorative_blocks             |1.7.0               |DONE      |NOSIGNATURE
		decorative_blocks_abnormals-1.16.4-1.1.jar        |Decorative Blocks Abnormals   |decorative_blocks_abnormals   |1.1                 |DONE      |NOSIGNATURE
		obfuscate-0.5.1-1.16.3.jar                        |Obfuscate                     |obfuscate                     |0.5.1               |DONE      |e1:59:1a:56:ec:97:b3:d0:b3:4b:25:06:1f:83:b0:f4:fd:0c:24:e3:6d:ea:94:b1:9f:22:b0:38:13:60:88:ea
		vehicle-mod-0.45.2-1.16.3.jar                     |MrCrayfish's Vehicle Mod      |vehicle                       |0.45.2              |DONE      |NOSIGNATURE
		structurize-0.13.100-RELEASE-universal.jar        |Structurize                   |structurize                   |0.13.100-RELEASE    |DONE      |NOSIGNATURE
		Bookshelf-1.16.4-9.3.18.jar                       |Bookshelf                     |bookshelf                     |9.3.18              |DONE      |ea:45:b3:82:b6:9d:50:16:95:e7:2e:34:e1:92:d5:b4:9b:69:90:d3:4f:2e:71:99:b0:be:40:80:27:1f:3e:b0
		Runelic-1.16.4-6.0.2.jar                          |Runelic                       |runelic                       |6.0.2               |DONE      |ea:45:b3:82:b6:9d:50:16:95:e7:2e:34:e1:92:d5:b4:9b:69:90:d3:4f:2e:71:99:b0:be:40:80:27:1f:3e:b0
		mcws-roofs-2.0.0-mc1.16.4.jar                     |Macaw's Roofs                 |mcwroofs                      |2.0.0               |DONE      |NOSIGNATURE
		DarkUtilities-1.16.4-7.0.3.jar                    |Dark Utilities                |darkutils                     |7.0.3               |DONE      |ea:45:b3:82:b6:9d:50:16:95:e7:2e:34:e1:92:d5:b4:9b:69:90:d3:4f:2e:71:99:b0:be:40:80:27:1f:3e:b0
		mcw-doors-1.0.1fix-mc1.16.4.jar                   |Macaw's Doors                 |mcwdoors                      |1.0.1               |DONE      |NOSIGNATURE
		furniture-7.0.0-pre19-1.16.3.jar                  |MrCrayfish's Furniture Mod    |cfm                           |7.0.0-pre19         |DONE      |NOSIGNATURE
		architectury-1.2.72-forge.jar                     |Architectury                  |architectury                  |1.2.72              |DONE      |NOSIGNATURE
		engineersdecor-1.16.4-1.1.5.jar                   |Engineer's Decor              |engineersdecor                |1.1.5               |DONE      |bf:30:76:97:e4:58:41:61:2a:f4:30:d3:8f:4c:e3:71:1d:14:c4:a1:4e:85:36:e3:1d:aa:2f:cb:22:b0:04:9b
		mcw-furniture-2.0.0-mc1.16.4.jar                  |Macaw's Furniture             |mcwfurnitures                 |2.0.0               |DONE      |NOSIGNATURE
		light-overlay-5.6.1.jar                           |Light Overlay                 |lightoverlay                  |5.6.1               |DONE      |NOSIGNATURE
		JEITweaker-1.16.4-1.0.1.5.jar                     |JEI Tweaker                   |jeitweaker                    |1.0.1.5             |DONE      |NOSIGNATURE
		cloth-config-forge-4.1.3.jar                      |Cloth Config v4 API           |cloth-config                  |4.1.3               |DONE      |NOSIGNATURE
		CraftTweaker-1.16.4-7.0.0.63.jar                  |CraftTweaker                  |crafttweaker                  |7.0.0.63            |DONE      |NOSIGNATURE
		ImmersivePetroleum-1.16.4-3.1.0-2.jar             |Immersive Petroleum           |immersivepetroleum            |3.1.0-2             |DONE      |NOSIGNATURE
		forge-1.16.4-35.1.32-universal.jar                |Forge                         |forge                         |35.1.32             |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		AppleSkin-mc1.16.2-forge-1.0.14.jar               |AppleSkin                     |appleskin                     |1.0.14              |DONE      |NOSIGNATURE
		BloodMagic-1.16.3-3.0.3-8.jar                     |Blood Magic                   |bloodmagic                    |1.16.3-3.0.3-8      |DONE      |NOSIGNATURE
		overloadedarmorbar-5.1.0.jar                      |Overloaded Armor Bar          |overloadedarmorbar            |5.1.0               |DONE      |NOSIGNATURE
		chiselsandbits-0.2.8-RELEASE.jar                  |Chisels & bits                |chiselsandbits                |NONE                |DONE      |NOSIGNATURE
		immersiveposts-1.16.3-3.0.0.jar                   |Immersive Posts               |immersiveposts                |3.0.0               |DONE      |d5:aa:49:67:b7:dd:64:8a:a4:7d:3e:57:12:6b:f9:3f:e8:5e:6b:24:d9:f9:c5:fb:c5:e7:a0:cf:98:64:dc:d0
		DynamicSurroundings-1.16.4-4.0.3.3.jar            |§3Dynamic Surroundings        |dsurround                     |1.16.4-4.0.3.3      |DONE      |b4:98:14:b9:76:55:25:4f:e5:5f:4d:71:90:87:43:5b:f9:d5:3a:02:60:42:5e:da:1f:15:9c:ff:be:a9:7c:77
	Crash Report UUID: 451abf4d-bb93-458c-b9e2-364844ca8890
	Patchouli open book context: n/a
	Launched Version: forge-35.1.32
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 1070/PCIe/SSE2 GL version 4.6.0 NVIDIA 457.51, NVIDIA Corporation
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: programer_art, vanilla, mod_resources
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-7400 CPU @ 3.00GHz

PS: looking at the crash log it appears it is somehow related to snow. I'm not an expert and tought this would be interesting.

**Comment by malte0811** [comment]
Duplicate of #4512. As a workaround you can disable smooth lighting to break the posts.

**Comment by HoovySimulator2** [comment]
oh, didn't find the other post when I searched for crash reports, sorry. I think this issue could be removed if it's not a problem?

**Comment by HoovySimulator2** [comment]
Found a good solution if you still want to use the posts!
The wooden posts from addon Immersive Posts by Twisted Gate doesn't crash if you build the exact same post that would have crashed with the default posts! And yes I did turn smooth lighting on to test to test this.
This could/should be added to the description of the mod if someone too is experiencing constant crashing with the default wooden posts. Hope this helps!
